config.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "API Sync",
 	"namespace": "Vanderbilt\\APISyncExternalModule",
-	"framework-version": 2,
+	"framework-version": 5,
```

Comment:
Just to make sure, were all the breaking changes on versions 3-5 considered?

Reply:
Yes.